File: Practice/src/exercise/FindPrimeNumbers.py
def solution(n):
    answer = 0
    arr = [True] * (n + 1)
    for i in range(2, n + 1):
        if arr[i]:
            answer += 1
            for j in range(i * i, n + 1, i):
                arr[j] = False
    return answer

    # num=set(range(2,n+1))
    # for i in range(2,n+1):
    #     if i in num:
    #         num-=set(range(2*i,n+1,i))
    # return len(num)

File: Practice/src/exercise/FindPrimeNumbers.js
function solution(n) {
    var answer = 0;
    const arr = Array(n + 1).fill(true);
    for (let i = 2; i <= n; i++) {
        if (arr[i]) {
            answer++;
            for (let j = i * i; j <= n; j *= i) {
                arr[j] = false;
            }
        }
    }
    return answer;
}
